spec.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: Semantic Specification
+---
+
+## Concepts and Terminology
+
+A **Trace** represents the potentially distributed, potentially concurrent data/execution path in a (potentially distributed, potentially concurrent) system.
+
+A **Span** represents a logical unit of work in the system that has a start time and an end time. Spans may be nested and ordered to model parent-child and casual relationships. A Trace can be thought of as a tree of Spans.
+
+Every Span has zero or more **Logs**, each of which being a timestamped message with an optional structured data payload of arbitrary size.
+
+Every Span may also have zero or more key/value **Tags**, which do not have timestamps and simply annotate the spans.
+
+A **Trace Context** encapsulates the smallest amount of state needed to describe a Span's identity within a larger, potentially distributed trace, sufficient to propagate the context of a particular trace between processes.
```

Comment:
I'd like to see more elaboration here as trace context is still a very abstract puzzle piece in my mind, so I think others would have trouble understanding from this description, too.

Is the trace context responsible for keeping track of the parent<->child span relationships? If so that'd be a good thing to explicitly mention here.